pkg/consts/platforms.go

```diff
@@ -0,0 +1,39 @@
+package consts
+
+import "fmt"
+
+type ClusterType string
+
+const (
+	ClusterTypeOpenshift  ClusterType = "openshift"
+	ClusterTypeKubernetes ClusterType = "kubernetes"
+)
+
+type ClusterFlavor string
+
+const (
+	ClusterFlavorVanillaK8s ClusterFlavor = "vanilla"
+	ClusterFlavorOpenshift  ClusterFlavor = "default"
+	ClusterFlavorHypershift ClusterFlavor = "hypershift"
+)
+
+// PlatformTypes
+type PlatformTypes int
```

Comment:
should we be consistent with the above consts and just have this as a string ? then we can drop the String() method ?